src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5542,19 +6159,16 @@ def create_functionapp(cmd, resource_group_name, name, storage_account, plan=Non
                                subnet_name=subnet_info["subnet_name"],
                                subnet_service_delegation=FLEX_SUBNET_DELEGATION if flexconsumption_location else None)
         subnet_resource_id = subnet_info["subnet_resource_id"]
-        vnet_route_all_enabled = True
```

Comment:
I don't have context on these changes - they don't seem related?

Reply:
The reason why I made these changes is because otherwise we get the "Readonly attribute name will be ignored in class" warning since vnet_route_all_enabled is only part of the SiteConfig class ([code ref](https://github.com/Azure/azure-sdk-for-python/blob/afce6c3a2c8c1c47688e8d0c896da9eac07477b6/sdk/appservice/azure-mgmt-web/azure/mgmt/web/v2024_11_01/models/_models_py3.py#L20941)) It is not part of the Site class: https://github.com/Azure/azure-sdk-for-python/blob/afce6c3a2c8c1c47688e8d0c896da9eac07477b6/sdk/appservice/azure-mgmt-web/azure/mgmt/web/v2024_11_01/models/_models_py3.py#L19638